app/views/attachments/_attachment_links.html.erb

```diff
@@ -12,7 +12,8 @@
         <% @document.attachments.each do |attachment| %>
             <li class="attachment">
               <span class="title"><%= attachment.title %></span>
-              <%= link_to "edit", edit_document_attachment_path(params[:document_type_slug], @document.content_id, attachment.content_id) %>
+              <%= link_to "edit", edit_document_attachment_path(params[:document_type_slug], @document.content_id,
```

Comment:
I would probably do like [this](https://gist.github.com/Rosa-Fox/37a0bf217c23736cec9c3ad84b9eda37) as separates it out a bit more.